torch/onnx/_internal/exporter/_compat.py

```diff
@@ -32,6 +32,7 @@ def _from_dynamic_axes_to_dynamic_shapes(
     model,
```

Comment:
```suggestion
    model,
    *,
```

Reply:
What is the benefit of doing this?

Reply:
This allows dynamic_axes to take a default argument while keeping output_names required.

Reply:
Oh I see. I still switched them